chart/templates/configmap.yaml

```diff
@@ -0,0 +1,37 @@
+{{- $namespace := .Values.common.namespace | required "A valid .config.namespace is required!" -}}
+{{- $yunikornHost := .Values.config.yunikornHost | required "A valid .config.yunikornHost is required!" -}}
+{{- $yunikornPort := .Values.config.yunikornPort | required "A valid .config.yunikornPort is required!" -}}
+{{- $yunikornProtocol := .Values.config.yunikornProtocol | required "A valid .config.yunikornProtocol is required!" -}}
+{{- $dbHost := .Values.config.dbHost | required "A valid .config.dbHost is required!" -}}
+{{- $dbPort := .Values.config.dbPort | required "A valid .config.dbPort is required!" -}}
+{{- $dbName := .Values.config.dbName | required "A valid .config.dbName is required!" -}}
+{{- $dbUser := .Values.config.dbUser | required "A valid .config.dbUser is required!" -}}
+{{- $dbPoolMaxConns := .Values.config.dbPoolMaxConns | required "A valid .config.dbPoolMaxConns is required!" -}}
+{{- $dbPoolMinConns := .Values.config.dbPoolMinConns | required "A valid .config.dbPoolMinConns is required!" -}}
+{{- $dbPoolMaxConnLifetime := .Values.config.dbPoolMaxConnLifetime | required "A valid .config.dbPoolMaxConnLifetime is required!" -}}
+{{- $dbPoolMaxConnIdleTime := .Values.config.dbPoolMaxConnIdleTime | required "A valid .config.dbPoolMaxConnIdleTime is required!" -}}
+{{- $yhsServerAddr := .Values.config.yhsServerAddr | required "A valid .config.yhsServerAddr is required!" -}}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: yhs-config
+  namespace: yunikorn
+  labels:
+    app: yunikorn-history-server
+    {{- with .Values.common.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+data:
```

Comment:
Shouldn't this contain a `yhs.yaml` or `config.yaml` similarly how we use the config here https://github.com/G-Research/yunikorn-history-server/blob/main/yhs.yml